src/main/java/org/example/homework12/Main.java

```diff
@@ -0,0 +1,66 @@
+package org.example.homework12;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.OptionalDouble;
+
+public class Main {
+    public static void main(String[] args) {
+
+        List<Student> students = createSampleData();
+
+        printAllStudentNames(students);
+        printNamesOfOlderStudents(students, 20);
+        printNamesOfAStudents(students);
+        printAverageGrade(students);
+        printHighestGradeStudent(students);
+        printCountBelow60(students);
+    }
+    private static List<Student> createSampleData() {
```

Comment:
better to add empty line between `}` and line with new method `private static ...`